### DISCUSSION TITLE
一直发邮箱,提示有人登入账号. | Keep sending emails, prompting that someone has logged into the account.

### DISCUSSION THREAD

**Discussion by g2112000290** [dani-garcia/vaultwarden#5048] [General]
一直发邮寄提示  说有人登入你的账号,其实我检查了下,没有人登入,这是什么原因,还有发送2步验证的时候,提示邮箱错误,但是实际是可以收到验证码的,也可以通过验证.
![搜狗截图20241005141240](https://github.com/user-attachments/assets/1e612792-3a62-4a6b-af17-9ba8a682762a)

Translated: 
I keep getting emails saying someone has logged into my account, but I checked and no one has logged in. What's the reason? Also, when I sent the 2-step verification, it said the email address was wrong, but I actually received the verification code and passed the verification.

**Comment by BlackDex** [comment]
Check the logs for more details. 

If you are trying to login, but you keep getting stuck on the 2FA part, this mail is valid. It warns you that someone or something was able to login with your password, but was not able to continue because the 2FA was not finished.
